jax/experimental/jax2tf/jax2tf.py

```diff
@@ -186,13 +211,19 @@ def convert(fun: Callable,
     enable_xla: if unset, the converter will try harder to use pure TF ops to
       convert the function, and raise an error if it can not be converted
       without resorting to XLA ops (default: True).
+    include_xla_op_metadata: if set, the generated TF ops will include
```

Comment:
If unset, will the TF ops have the same XLA op metadata (after the action of the bridge) that they'd have if they were written in raw TF, or will they have no metadata? And does this one flag also control non-XLA metadata in the form of the TF graph name scope?

Reply:
I have refined the comment to say that if unset then TF will use default OpMetadata (as if written in raw TF). Those will not contain correct source location, because TF will pick the top-most non-TF stack trace, and that would
point to some place in the jax2tf converter. Furthermore, without these OpMetadata, when loading from SavedModel the source location will point to the line where `tf.saved_model.load()` is called. 

The name scopes are not affected by this flag.